### DISCUSSION TITLE
Error: diff failed: duplicate endpoint . You may add the x-since-date extension to specify order

### DISCUSSION THREAD

**Discussion by Celldweler** [Tufin/oasdiff#577] [Q&A] [answered]
So I encountered an issue while trying to compare two specifications. I received the following error, and I couldn't find any solutions online.

``>.\oasdiff.exe diff .\base.json .\revision.json``

  `` Error: diff failed: duplicate endpoint (GET /api/tasks/{taskId}/attributes/{name}) found in .\base.json and .\base.json. You may add the x-since-date extension to specify order``

 Interestingly, I was able to compare the same files successfully using the tool at https://www.oasdiff.com/diff-calculator.

Could you please help me understand what might be causing this problem? Are there any specific flags that need to be set before running the diff command? I'm using the local version on my windows machine.

Thank you.

![image](https://github.com/Tufin/oasdiff/assets/55654940/861e2ee6-4fac-40b9-a350-ca5323fb1822)

Oasdiff online diff calculator output:

![image](https://github.com/Tufin/oasdiff/assets/55654940/c29c5595-dc00-4d10-aa51-ae4f935b77a5)

**Comment by reuvenharrison** [comment, marked as answer]
Please read https://github.com/Tufin/oasdiff/blob/main/docs/MATCHING-ENDPOINTS.md
If you still need help please provide the specs you are comparing.
